<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_block</class>
 <widget class="QWidget" name="form_block">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="lbl_block">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[New/Test Block #]</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_widget_block">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_cases">
      <attribute name="title">
       <string>Block Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="hlayout_id">
         <item>
          <widget class="QLabel" name="lbl_id">
           <property name="text">
            <string>Id:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_id">
           <property name="placeholderText">
            <string>[Identificator]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_name">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_name">
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>[Name]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_system">
           <property name="text">
            <string>System:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_system"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_date">
         <item>
          <widget class="QLabel" name="lbl_creation">
           <property name="text">
            <string>Creation time:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_creation">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>[Creation time]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_update">
           <property name="text">
            <string>Last update:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_update">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>[Update time]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_comments">
         <item>
          <widget class="QLabel" name="lbl_comments">
           <property name="text">
            <string>Comments:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_comments">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>[Comments]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_info">
      <attribute name="title">
       <string>Cases</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tbl_cases">
         <property name="rowCount">
          <number>4</number>
         </property>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Assets</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Comments</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Added</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Last updated</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hlayout_btn_case">
         <property name="leftMargin">
          <number>450</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_add_case">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_remove_case">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_btn_block">
     <property name="leftMargin">
      <number>400</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_accept">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
